dev/ci/ci-user-overlay.sh

```diff
@@ -29,3 +30,8 @@ if [ $TRAVIS_PULL_REQUEST == "461" ] || [ $TRAVIS_BRANCH == "stm+remove_compat_p
     mathcomp_CI_BRANCH=no_camlp4_compat
     mathcomp_CI_GITURL=https://github.com/ejgallego/math-comp.git
 fi
+
+if [ $TRAVIS_PULL_REQUEST == "565" ] || [ $TRAVIS_BRANCH == "remove-vernacerror" ]; then
+    mathcomp_CI_BRANCH=remove-vernacerror
+    mathcomp_CI_GITURL=https://github.com/SkySkimmer/math-comp.git
```

Comment:
When I use those paths I need to do a special fetch command and then checkout FETCH_HEAD.
Just setting CI_BRANCH won't do it.